### PR TITLE
Feature/childcare inspections

### PR DESCRIPTION
**Pull request by thy-neighbor**

### CODE REVIEW

File: childcare/entity.py [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+            {'field': 'legalname', 'column': 'legal_name'},
+            {'field': 'zipcode', 'column': 'zipcode_id', 'data': self.get_zipcode_id},
+            {'field': 'status'},
+            {'field': 'dc_id', 'column': 'public_id'},
+            {'field': 'childcaretype', 'column': 'childcare_type_id', 'data': self.get_caretype_id},
+        ]
+        self.cacheable_fields = ['public_id']
+
+    def skip_record(self, record):
+        return self.record_cache and record['dc_id'] in self.record_cache
```
Comment (CThaw90) [inline childcare/entity.py]: `dc_id` doesn't seem to be unique per childcare center we will probably need to use the center_name column to identify a unique center. We will also need to put a unique index on that column as well, there are a lot of duplicates in the database.

File: childcare/entity.py [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+        while continue_fetching:
+            request_url = API_URL.format(
+                begin_date.isoformat(),
+                today_date.isoformat(),
+                offset
+            )
+            records = json.loads(requests.request('GET', request_url).content.decode('utf-8'))
+            
+            for item in records:
+                if 'zipcode' in item:
```
Comment (CThaw90) [inline childcare/entity.py]: You can combine these two nested if statements `if 'zipcode' in item and item['dc_id'] not in dc_id_set:`

File: childcare/entity.py [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+    
+    def to_lowercase(self, record, field):
+        return record[field].lower()
+    
+    def __init__(self):
+        super().__init__()
+
+        self.table_name = 'childcare_center'
+        self.fields = [
+            {'field': 'centername', 'column': 'center_name', 'data': self.to_lowercase},
```
Comment (CThaw90) [inline childcare/entity.py]: Same here we don't want to save the value to database as lowercase.

File: childcare/entity.py (hunk 2 above)
Comment (thy-neighbor) [reply childcare/entity.py]: Just now seeing this, will do

**Review by CThaw90 — APPROVED** [review]

**Pull request merged**